lib/Swagger/src/Discussion/Models/PostInput.php

```diff
@@ -63,15 +63,16 @@ class PostInput implements ArrayAccess
       * @var string[]
       */
     protected static $swaggerTypes = array(
-        'body' => 'string',
-        'thread_id' => 'int',
+        'id' => 'int',
         'position' => 'int',
-        'title' => 'string',
         'site_id' => 'int',
-        'open_graph' => '\Swagger\Client\Discussion\Models\OpenGraphInput',
-        'creator_id' => 'int',
         'deleted_position' => 'int',
-        'id' => 'int'
+        'open_graph' => '\Swagger\Client\Discussion\Models\OpenGraph',
+        'content_images' => '\Swagger\Client\Discussion\Models\ContentImage[]',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "content_images" 9 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)